File: C/Esercizi/0_Eserciziario/es09.c
/********************************************
LIVELLO: Facile
Scuola: Liceo, Secondo Anno
Conoscenza: Scanf, Printf
*********************************************/

/*
9. Scrivere un algoritmo che, dato il prezzo di un prodotto, calcoli il prezzo scontato del 20%.
*/

#include <stdio.h>

int main(){ 
  float a;
  float b;
  printf ("inserire prezzo del prodotto");
  scanf ("%f", &a);
  b = a - (20*a)/100;
  printf ("prezzo scontato=%f", b);

	return 0;
}

File: Java/Esercizi/0_Eserciziario/es09.java
// Scrivere un algoritmo che, dato il prezzo di un prodotto, calcoli il prezzo scontato del 20%.

public class es09 {	

	public static void main(String[] args) {
		
		int prezzo_base = 50;
    int prezzo_scontato = prezzo_base - (prezzo_base*20/100);
    
    System.out.println(prezzo_scontato);
	}

}
